docs/Metrics/DeployTime.md

```diff
@@ -16,11 +16,26 @@ The time it takes from when a PR is merged to when it is deployed.
 
 
 ## How is it calculated?
-You can define `deployment` based on your actual practice. For a full list of `deployment`'s definitions that DevLake support, please refer to [Deployment Frequency](/docs/Metrics/DeploymentFrequency.md).
+1. You can define `deployment` based on your actual practice. For a full list of `deployment`'s definitions that DevLake support, please refer to [Deployment Frequency](/docs/Metrics/DeploymentFrequency.md).
```

Comment:
There's no related section about `deployment` list in 'Deployment Frequency' doc.

Text change: 
`PR deploy time` is calculated by subtracting a PR's deployed_date and merged_date. Hence, we should associate PR/MRs with `deployments`. 

Here are the detailed steps:
1. Find the [commits diff](the link to refdiff plugin) between two consecutive deployments by deployments' commit_sha under the same scope and environment (in terms of TESTING/STAGING/PRODUCTION)
2. ....
3. ....